plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/index.ts

```diff
@@ -1,6 +1,7 @@
 import { ComponentType } from "react";
 
 // general
+import ButtonLink from "./general/ButtonLink";
```

Comment:
It is really small, but can you try to keep order the same (at least in the same file). It is just a small thing but will allow our brains to focus less on finding things and more on other things.
So the // general import order = `fields` order = FieldComponent type order etc.

Reply:
sorry I didn't get te idea , as some times especially in the import part , it is auto order , do you mean in type definition

Reply:
30294f5aa82bcbb0ea19d6d665ee117a42cea95a